test/framework/src/main/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertions.java

```diff
@@ -669,6 +670,11 @@ public static void assertVersionSerializable(Version version, Streamable streama
                     equalTo(0));
             assertThat("Serialization failed with version [" + version + "] bytes should be equal for streamable [" + streamable + "]",
                     serialize(version, streamable), equalTo(orig));
+        } catch (UnsupportedOperationException ex) {
+            if (ex.equals("can't read named writeable from StreamInput")) {
```

Comment:
They are serializable, but we can't do the round tripping of aggregations without NamedWriteableRegistry.